index.html

```diff
@@ -1,311 +1,210 @@
-          </div>
+          <ul>
+            <li>Bootstrap</li>
+            <li>Ruby on Rails</li>
+            <li>RSpec</li>
+            <li>Capybara</li>
+            <li>Selenium</li>
+          </ul>
         </div>
-
-        <div class="footer-card">
-          <div>
-            <img class="footer-img" src="./images/icon-skills.svg" alt="" />
-          </div>
+        <div class="about-card">
+          <img src="./images/icon-skills.svg" alt="">
           <h2>Skills</h2>
-          <div class="group-tag">
-            <div class="footer-list">
-              <h4 class="footer-tags">codekit</h4>
-              <h4 class="footer-tags">Github</h4>
-              <h4 class="footer-tags">Codepen</h4>
-            </div>
-            <div class="footer-list">
-              <h4 class="footer-tags">Gitlab</h4>
-              <h4 class="footer-tags">Terminal</h4>
-            </div>
-          </div>
+          <ul>
+            <li>Codekit</li>
+            <li>Github</li>
+            <li>Codepen</li>
+            <li>Gitlab</li>
+            <li>Terminal</li>
+          </ul>
         </div>
       </div>
     </section>
 
     <section>
-      <h1 class="footer-header">
-        I'm always interested in hearing about new projects, so if you'd like to
-        chat please get in touch.
-      </h1>
-      <form class="contact-form">
-        <lable for="fname"></lable>
-        <input
-          type="text"
-          id="fname"
-          name="firstname"
-          placeholder="Full name"
-        />
-
-        <label for="email"></label>
-        <input
-          type="text"
-          id="email"
-          name="email"
-          placeholder="Email Address"
-        />
-
-        <label for="subject"></label>
-        <textarea
-          id="subject"
-          name="subject"
-          placeholder="Enter Text Here"
-        ></textarea>
-
-        <input type="submit" value="Get in Touch" />
-      </form>
-
-      <div>
-        <ul class="follow-link">
-          <li>
-            <a href="https://twitter.com"
-              ><img src="./images/twitter.svg" alt=""
-            /></a>
-          </li>
-          <li>
-            <a href="https://linkedin.com"
-              ><img src="./images/Linkedin icon.svg" alt=""
-            /></a>
-          </li>
-          <li>
-            <a href="https://medium.com"
-              ><img src="./images/medium.svg" alt=""
-            /></a>
-          </li>
-          <li>
-            <a href="https://github.com"
-              ><img src="./images/github.svg" alt=""
-            /></a>
-          </li>
-          <li>
-            <a href="https://angellist.com"
-              ><img src="./images/angellist.svg" alt=""
-            /></a>
-          </li>
+      <footer class="contact-heading">
+        <h1>I'm always interested in hearing about new projects, so if you'd like to chat 
+          please get in touch.
+        </h1>
+        <form action="" class="contact-form">
```

Comment:
- [ ] Dear @Mekuseo, Your form is looking great already. Please remember this form should uses the `action` and `method` attribute to integrate with [formspree service](https://formspree.io/html).
> * `action` value should be similar to `https://formspree.io/f/{form_id}`
> * `method` value equals `POST`
- [ ] Also, When the user clicks on submit form should be successfully saved to the `formspree`

Reply:
Have added the action and method.
Thank you for letting talking about form spree. This is my first time hearing it and had to sign up. Looks beautiful.